src/steps/then.js

```diff
@@ -35,5 +36,21 @@ module.exports = function () {
             }
         });
 
+        Then(_r(`^${pageObject} has text ${dictionaryObject}$`), async function async (element, textPhrase) {
+            /**
+            * @param {pageObject} element - String or "page"."object" to select the element
+            * @type {String} or {DictionaryObject}
+             */
+            const phrase = getDictionaryObject.call(this, textPhrase);
+            const locator = getPageObject(element);
+            const locatorText = browser.getText(locator);
+
+            try {
+                return await locatorText === phrase;
```

Comment:
Text comparing is a not async step so it should not be marked as await. Also, I beleive that `expect` is better for strings comparing and we can use it instead of `try - catch` here

Reply:
Done